### PR TITLE
app-misc/pax-utils: Add prefix support

### PR DESCRIPTION
**Pull request by wiredhikari**
Signed-off-by: Atharva <atharvaamritkar@protonmail.com>

### CODE REVIEW

File: app-misc/pax-utils/pax-utils-1.3.3.ebuild [hunk 1]
```diff
@@ -68,7 +68,7 @@ src_test() {
 }
 
 src_install() {
-	_emake DESTDIR="${D}" PKGDOCDIR='$(DOCDIR)'/${PF} install
+	_emake DESTDIR="${ED}" PKGDOCDIR='$(DOCDIR)'/${PF} install
```
Comment (amadio) [inline app-misc/pax-utils/pax-utils-1.3.3.ebuild:71]: Please drop this change, 1.3.3 is not affected. I just installed it on my prefix without this change and all files get installed into the right places already.

**Comment by thesamesam** [comment]
Feel free to merge once happy, @amadio.

**Comment by amadio** [comment]
Thanks, @thesamesam. I'm aware of this, just was too busy to test it, but should be possible to do it today.

**Comment by amadio** [comment]
This doesn't seem to be needed. I just emerged `app-misc/pax-utils-1.3.4` without this change and it went just fine on prefix:
```sh
gentoo-prefix ~ $ equery l pax-utils
 * Searching for pax-utils ...
[IP-] [  ] app-misc/pax-utils-1.3.4:0
gentoo-prefix ~ $ qlist pax-utils
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/share/man/man1/scanmacho.1.bz2
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/share/man/man1/scanelf.1.bz2
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/share/man/man1/pspax.1.bz2
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/share/man/man1/dumpelf.1.bz2
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/scanmacho
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/pspax
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/dumpelf
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/scanelf
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/symtree
/cvmfs/sft.cern.ch/lcg/contrib/gentoo/linux/usr/bin/lddtree
```

**Pull request closed without merging**